### PR TITLE
(PDOC-33) Inform reader how to install yard gem

### PR DESCRIPTION
**Pull request by iankronquist**

### CODE REVIEW

**Comment by MikaelSmith** [comment]
I believe something like `puppet resource puppet_gem yard ensure=present` would also work. I'll give it a try.

File: README.md [hunk 1]
```diff
@@ -32,14 +32,17 @@ In order to run strings you need to have the following software installed:
   * Puppet 3.7 or newer
   * The yard rubygem
 
-Install the yard rubygem with the gem command:
+Install the yard rubygem to Puppet's Gem set.
```
Comment (kylog) [inline README.md]: I might just reduce this to 'Install the yard rubygem' (and leave out the prepositional phrase)

File: README.md [hunk 2]
```diff
@@ -32,14 +32,17 @@ In order to run strings you need to have the following software installed:
...
 
+On *nix systems, run:
 ```
-$ gem install yard
```
Comment (kylog) [inline README.md]: And as I mentioned in hipchat, maybe have a section for installing the gem on 3.7+ and a separate snippet for 4.x.

**Comment by iankronquist** [comment]
@MikaelSmith Does that work for Puppet 3.x? I see a matrix here:


|                  | 3.x                | 4.x                           |   |   |
|------------------|--------------------|-------------------------------|---|---|
| Windows with gem | gem install yard?? | gen with different awful path |   |   |
|  * nix with gem  | gem install yard   | gem with awful opt path       |   |   |
| Using puppet     | no?                |                               |   |   |

**Comment by MikaelSmith** [comment]
No, puppet_gem provider is new in 4.x

**Comment by iankronquist** [comment]
@kylog @MikaelSmith @HAIL9000 Thoughts on the latest revision?

**Comment by MikaelSmith** [comment]
Works for me.

File: README.md [hunk 3]
```diff
@@ -53,6 +63,7 @@ Simply run this command and you're off to the races:
 $ puppet module install puppetlabs-strings
 ```
 
+
```
Comment (kylog) [inline README.md]: You could drop this blank line.

**Comment by kylog** [comment]
I'd squash the commits. Also, including what we *didn't* do in the commit message seems overkill and confusing.

Other than that, lgtm. I like specifying it in terms of operations with `puppet` :)

File: README.md (hunk 3 above)
Comment (iankronquist) [inline README.md]: Fixed.

**Comment by kylog** [comment]
Woot!

**Pull request merged**